src/config/services/configuring.md

```diff
@@ -5,28 +5,13 @@ options can be set via a `conf` file in the service directory. This allows
 service customization without modifying the service directory provided by the
 relevant package.
 
+Check the service file for how to pass configuration parameters, but services
+usually expect `OPTS=--value1 --value2` as values.
+
 To make more complex customizations than provided by default, edit the service.
 
 ## Editing services
 
-To edit a service, first duplicate its service directory under a different name:
-
-```
-# cp -aR /etc/sv/service_name /etc/sv/service_name_edited
-```
-
-The contents of the duplicated directory can then be edited as needed, for
-example, by using `vi(1)`:
-
-```
-# vi /etc/sv/service_name_edited
-```
-
-After editing, the old service should then be stopped, disabled and replaced
-with the new one:
-
-```
-# sv down service_name
-# rm /var/service/service_name
-# ln -s /etc/sv/service_name_edited /var/service/
-```
+To edit a service, first duplicate its service directory under a different name.
+Edit the new service file as needed. After editing, the old service should then
+be stopped, disabled and replaced with the new service.
```

Comment:
I think it'd be better if you explained that this one is necessary because the `run` file will be replaced by the package manager, so editing it is not enough.

And the phrases were separated because they had commands between them, so perhaps they can flow better? Something like

```
To edit a service, first duplicate its service directory under a different name. Then, edit the new service file as needed. Finally, the old service should then be stopped, disabled and replaced with the new service.
```